homeassistant/components/spc/__init__.py

```diff
@@ -62,19 +79,27 @@ async def async_upate_callback(spc_object):
         _LOGGER.error("Failed to load area/zone information from SPC")
```

Comment:
This should raise [ConfigEntryNotReady](https://github.com/home-assistant/core/blob/dev/homeassistant/exceptions.py#L119)